support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -155,11 +163,23 @@ object DigitalPackValidation {
         currencyIsSupportedForCountry(country, currency) and
         PaidProductValidation.noEmptyPaymentFields(paymentFields)
 
+    def isInBenefitsTest(supportAbTests: Set[AbTest]) = {
+      if (
+        isValidBenefitsTestPurchase(
+          createSupportWorkersRequest.product.asInstanceOf[DigitalPack],
+          Some(createSupportWorkersRequest.supportAbTests),
+        )
+      ) Valid
+      else
+        Invalid("User is not in the benefits test")
```

Comment:
The thing is that this will be the case for anyone not in the variant of this test so is not actually an error condition, we are just checking that if they ARE in the test then certain conditions have been met

Reply:
late comment
I see what you mean that this is almost a pre-check rather than the actual validation.
It is indeed a good idea to avoid the "try this, and if it fails try this" as the failure messages get super comples.
However in this case if the purchase isn't detected as in the test when it should be, then I would like to know the exact reason why it thought they weren't in the benefits test.